google-cloud-spanner/src/main/java/com/google/cloud/spanner/connection/ConnectionOptions.java

```diff
@@ -213,8 +214,8 @@ public String[] getValidValues() {
   public static final String LENIENT_PROPERTY_NAME = "lenient";
   /** Name of the 'rpcPriority' connection property. */
   public static final String RPC_PRIORITY_NAME = "rpcPriority";
-  /** Dialect to use for a connection. */
-  private static final String DIALECT_PROPERTY_NAME = "dialect";
```

Comment:
I agree that we should remove this, as it is no longer needed, BUT:
1. We should consider doing so in a separate PR. It is not related to this change.
2. We should do it in a non-breaking way. Anyone who currently has `dialect=...` in their connection string will get an error after this change. We should therefore silently ignore it if we encounter `dialect=...` in a connection string.

Reply:
Reverted the change to remove Dialect, will be done in another PR